src/hooks/useClaimAll.ts

```diff
@@ -9,7 +9,8 @@ import { getIndividualClaimTxs } from './helper/claim';
 import { useCurrentEra, useCustomSignature } from './index';
 
 export function useClaimAll() {
-  const batchTxs = ref<ExtrinsicPayload[]>([]);
+  let batchTxs: ExtrinsicPayload[] = [];
```

Comment:
Could you share with me why you are using `let` instead of `ref`?

Reply:
Sure. Ref is a proxy object and I stumbled upon following issue https://github.com/polkadot-js/api/issues/4656#issuecomment-1072068847